Profiling: Add docs for Parca and Phlare (#57833)
grafana/grafana: docs/sources/datasources/phlare.md

---
aliases:
- /docs/grafana/latest/datasources/phlare/
- /docs/grafana/latest/features/datasources/phlare/
  description: Horizontally-scalable, highly-available, multi-tenant continuous profiling aggregation system. OSS profiling solution from Grafana Labs.
  keywords:
- grafana
- phlare
- guide
- profiling

>   title: Phlare

Reviewer: ```suggestion
  title: Phlare data source
```

Author: Actually did not change it after all. In https://grafana.com/docs/grafana/latest/datasources/ we show it on the left under Datasources header so it looks weird to repeat that after each name. We have that right now only with influx not sure if accidental or not.

  weight: 1150
---

# Phlare data source

> **Note:** This feature is behind the `flameGraph` feature toggle.
> You can enable feature toggles through configuration file or environment variables. See configuration [docs]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}) for details.
> Grafana Cloud users can access this feature by [opening a support ticket in the Cloud Portal](https://grafana.com/profile/org#support).

> Grafana ships with built-in support for Phlare, a horizontally scalable, highly-available, multi-tenant, OSS, continuous profiling aggregation system from Grafana Labs. Add it as a data source, and you are ready to query your profiles in [Explore]({{< relref "../explore" >}}).

Reviewer: ```suggestion
Grafana ships with built-in support for Phlare, a horizontally scalable, highly-available, multi-tenant, OSS, continuous profiling aggregation system from Grafana Labs. When you install the Phlare data source, you can query your profiles in [Explore]({{< relref "../explore" >}}).
```

Author: Same as for Parca as this is in Grafana already no need to install it.

> ## Add data source

Reviewer: Same general comments as Parca
